core/src/test/java/org/mskcc/oncotree/utils/OncotreeTestConfig.java

```diff
@@ -334,650 +333,650 @@ private List<String> setupExpectedMainTypeList() throws Exception {
-                "VGCE", "Villoglandular Adenocarcinoma of the Cervix", "Cervical Cancer", "Teal", "CERVIX", "http://data.mskcc.org/ontologies/oncotree/ONC000913",
-                "VGCT", "Germ Cell Tumor of the Vulva", "Germ Cell Tumor", "Purple", "VULVA", "http://data.mskcc.org/ontologies/oncotree/ONC000914",
-                "VIMT", "Immature Teratoma", "Germ Cell Tumor", "Purple", "VGCT", "http://data.mskcc.org/ontologies/oncotree/ONC000915",
-                "VMA", "Mucinous Adenocarcinoma of the Vulva/Vagina", "Vulvar Carcinoma", "Purple", "VULVA", "http://data.mskcc.org/ontologies/oncotree/ONC000916",
-                "VMGCT", "Mixed Germ Cell Tumor", "Germ Cell Tumor", "Purple", "VGCT", "http://data.mskcc.org/ontologies/oncotree/ONC000917",
-                "VMM", "Mucosal Melanoma of the Vulva/Vagina", "Melanoma", "Purple", "VULVA", "http://data.mskcc.org/ontologies/oncotree/ONC000918",
-                "VMT", "Mature Teratoma", "Germ Cell Tumor", "Purple", "VGCT", "http://data.mskcc.org/ontologies/oncotree/ONC000919",
-                "VOEC", "Embryonal Carcinoma", "Germ Cell Tumor", "Purple", "VGCT", "http://data.mskcc.org/ontologies/oncotree/ONC000920",
-                "VPDC", "Poorly Differentiated Vaginal Carcinoma", "Vaginal Cancer", "Purple", "VULVA", "http://data.mskcc.org/ontologies/oncotree/ONC000921",
-                "VPE", "Polyembryoma", "Germ Cell Tumor", "Purple", "VGCT", "http://data.mskcc.org/ontologies/oncotree/ONC000922",
-                "VPSCC", "Verrucous Penile Squamous Cell Carcinoma", "Penile Cancer", "Blue", "PSCC", "http://data.mskcc.org/ontologies/oncotree/ONC000923",
-                "VSC", "Squamous Cell Carcinoma of the Vulva/Vagina", "Vaginal Cancer", "Purple", "VULVA", "http://data.mskcc.org/ontologies/oncotree/ONC000924",
-                "VULVA", "Vulva/Vagina", null, "Purple", "TISSUE", "http://data.mskcc.org/ontologies/oncotree/ONC000925",
-                "VYST", "Yolk Sac Tumor", "Germ Cell Tumor", "Purple", "VGCT", "http://data.mskcc.org/ontologies/oncotree/ONC000926",
-                "WDLS", "Well-Differentiated Liposarcoma", "Soft Tissue Sarcoma", "LightYellow", "LIPO", "http://data.mskcc.org/ontologies/oncotree/ONC000927",
-                "WDTC", "Well-Differentiated Thyroid Cancer", "Thyroid Cancer", "Teal", "THYROID", "http://data.mskcc.org/ontologies/oncotree/ONC000928",
-                "WM", "Waldenstrom Macroglobulinemia", "Non-Hodgkin Lymphoma", "LimeGreen", "BCL", "http://data.mskcc.org/ontologies/oncotree/ONC000929",
-                "WPSCC", "Warty Penile Squamous Cell Carcinoma", "Penile Cancer", "Blue", "PSCC", "http://data.mskcc.org/ontologies/oncotree/ONC000930",
-                "WT", "Wilms' Tumor", "Wilms Tumor", "Orange", "KIDNEY", "http://data.mskcc.org/ontologies/oncotree/ONC000931"};
+                "AA", "Aggressive Angiomyxoma", "Soft Tissue Sarcoma", "LightYellow", "SOFT_TISSUE", "http://data.mskcc.org/ontologies/oncotree/ONC000288", "ONC000288",
```

Comment:
The field value which comes before the added (final) field seems to be the URI for the concept. However, these URIs are going to be invalid under graphite (The new URIs contain UUID instead).
Maybe that entire field can be eliminated? I don't think we want to rewrite each one with the appropriate UUID.

above we are setting the URI with :
nextNode.setURI(rawTestValueSource[pos + 5]);

we should look to see if this is ever referenced in the test code. I don't see any references to:
rawTestValueSource[pos + 5] in any of the "setup" functions here. So maybe we can just drop that field.

Reply:
field was not dropped, but was shortened to just the unique code (subset identifier)